src/library/scala/sys/process/package.scala

```diff
@@ -231,6 +231,7 @@ package scala.sys {
       type OutputStream           = java.io.OutputStream
       type SyncVar[T]             = scala.concurrent.SyncVar[T]
       type URL                    = java.net.URL
+      type Seq[A]                 = scala.collection.Seq[A]
```

Comment:
I don’t think we should go that way. Multiplying the number of `Seq` aliases might only multiply the sources of confusion.

Reply:
Agreed!

Reply:
Would you prefer the full-qualifying approach? The advantage of the alias is of course that you reduce the risk of forgetting to use the full-qualified on some places.

I am fine with both solutions.

Reply:
To me, using `scala.collection.Seq` looks OK. I'm also fine importing `scala.collection.Seq` in the top of the file(s).

Reply:
Yes, I think using fully qualified names is better here.

Reply:
If nobody objects I will change it back to full qualified names tomorrow then.

Thanks

Reply:
Thanks Alex!

Reply:
It seems to me that if the package offers one-stop shopping for all its aliases, then that's the right place for the change. (Even if I wouldn't write it that way in the first place.) The very fact that the previous edit failed to "hit all the Seqs" says that doing it here is better. Once the code is spun off, it can get rewritten for greater clarity.

Reply:
I can see both your points, I understand that seems weird to add a type alias called Seq and I also understand that adding this type alias when there are already several seems to keep it consistent with the current code. So as there is different opinions I would lie to know how do we proceed here?

Another possibility would be to give this alias a more specific name to reduce confusion.

Reply:
@alex1712 Your first approach of correcting the references is OK and preferred by majority. I just wanted my opinion in the record.

Reply:
I would change it then, thank you all very much.